code/modules/reagents/reagents/food-Drinks.dm

```diff
@@ -188,8 +191,10 @@
 	taste_mult = 2
 	reagent_state = LIQUID
 	nutriment_factor = 0.8
+	sanity_gain_ingest = 0 //Yucky
```

Comment:
make the worst of these 0-gain ingests actually remove sanity instead, no balls

Reply:
I actually agree. Eating shit food should make you feel like shit.

Reply:
@heymando Heck you, I have plenty of balls